### PR TITLE
wright fisher text

### PR DESCRIPTION
**Pull request by molpopgen**
Add prose related to the default/implicit model that demes has in mind.

### CODE REVIEW

File: paper.tex [hunk 1]
```diff
@@ -257,6 +257,29 @@ \subsection*{Population genetics model}
...
+\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+	I'm not making an attempt yet to revise that later content--I think other authors should
+	weigh in.}
+
+\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+
+Within a population, we implicitly assume that allele frequency dynamics can
+be described by the "Wright-Fisher" model.  Briefly, generations are non-overlapping (all parents
+reproduce and die simultaneously) and the expected frequency (at birth) of an allele currently at frequency $p_i$
+one generation in the future is $p_iw_i/\bar{w}$, where $w_i$ and $\bar{w}$ are the marginal and
```
Comment (apragsdale) [inline paper.tex]: maybe "in the next generation" instead of "one generation in the future"

File: paper.tex [hunk 2]
```diff
@@ -257,6 +257,29 @@ \subsection*{Population genetics model}
...
+mean fitnesses, respectively, properly weighted according to ancestry proportions. In this framework,
+a forward-time simulation of finite populations is equivalent to multinomial sampling of allele frequencies each
+generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970})
+and a backwards-time (coalescent) simulation follows the approximations described in \cite[][chapter 3]{wakeley2008-hd}.
+Further, this model assumes "soft" selection \citep{christiansen1975hard}, meaning that the dynamics of population sizes
+changes are \textit{independent} of the details of inidvidual fitnesses.
+Numerous examples of both types of simulators using this model currenly exit (REFS), each with their own unique method
+of specifying the demography of a simulation.
```
Comment (apragsdale) [inline paper.tex]: Do you mean "specifying the details of selection in a simulation"?

**Review by apragsdale — APPROVED** [review]
Looks really good to me! Just a small comment and question

File: paper.tex (hunk 2 above)
Comment (molpopgen) [reply paper.tex]: I can go either way on the wording here.

**Review by apragsdale — APPROVED** [review]
This still looks good to me!

File: paper.tex [hunk 3]
```diff
@@ -257,6 +257,32 @@ \subsection*{Population genetics model}
...
 
+\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+	I'm not making an attempt yet to revise that later content--I think other authors should
+	weigh in.}
+
+\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+
+Within a population, we implicitly assume that allele frequency dynamics can
+be described by the "Wright-Fisher" model.  Briefly, generations are non-overlapping (all parents
+reproduce and die simultaneously) and the expected frequency (at birth) of an allele currently at frequency $p_i$
```
Comment (grahamgower) [inline paper.tex]: s/an allele/an allele $i$/ ?

File: paper.tex [hunk 4]
```diff
@@ -257,6 +257,32 @@ \subsection*{Population genetics model}
...
+one generation in next generation is $p_iw_i/\bar{w}$, where $w_i$ and $\bar{w}$ are the marginal and
+mean fitnesses, respectively, properly weighted according to ancestry proportions. In this framework,
+a forward-time simulation of finite populations is equivalent to multinomial sampling of allele frequencies each
+generation (\citet[][pp 29-31]{burger2000-ul}, \citet[][pp 179-181]{crowkimura1970})
```
Comment (grahamgower) [inline paper.tex]: I'm confused. How are the fitness effects taken into account if one just does multinomial sampling of allele frequencies?

Comment (molpopgen) [reply paper.tex]: Using E[pi] as defined as the coefficients. The slow way to do a sim is to calculate marginal allele fitnesses and then call rmultinom(ps, N).

Comment (molpopgen) [reply paper.tex]: At birth, f(AA) i p^2.  After selection, it is f(AA)' = p^2wAA/wbar.  Likewise f(Aa)' = 2pqwAa/wbar.  E[p] at time t + 1 is f(AA)' + f(Aa)'/2, etc., and it has a binomial number of offspring.We don't want to write all this as it is in the citations.

File: paper.tex [hunk 5]
```diff
@@ -257,6 +257,32 @@ \subsection*{Population genetics model}
...
 and end sizes that must remain positive.
 
+\krtcomment{The paragraph(s) below subsume(s)/replace(s) some of what appears in later pargraphs.
+	I'm not making an attempt yet to revise that later content--I think other authors should
+	weigh in.}
+
+\krtcomment{Aaron -- I have perhaps over-simplified the description of marginal allele frequency dynamics?}
+
+Within a population, we implicitly assume that allele frequency dynamics can
+be described by the "Wright-Fisher" model.  Briefly, generations are non-overlapping (all parents
```
Comment (jeromekelleher) [inline paper.tex]: latex quotes here like ``quote''. Probably don't need scare quotes on WF though

**Review by jeromekelleher — COMMENTED** [review]
I like it! Giving just enough detail to relate to existing texts is the right approach I think.

I say we merge this and then loop back to correct other stuff in the document.

File: paper.tex (hunk 5 above)
Comment (molpopgen) [reply paper.tex]: I'll fix that.  One of the latex gotchas that always gets me. 🙄

**Pull request merged**